awsim_sensor_kit_launch/config/concatenate_and_time_sync_node.param.yaml

```diff
@@ -0,0 +1,22 @@
+/**:
+  ros__parameters:
+    debug_mode: false
+    has_static_tf_only: false
+    rosbag_replay: false
+    rosbag_length: 20.0
+    maximum_queue_size: 5
+    timeout_sec: 0.2
```

Comment:
I checked `sensor_points_delay_time_sec`, which means the time elapsed from the sensor to reaching `autoware_ndt_scan_matcher`.

* Before (11~14msec)

![Screenshot from 2024-11-08 17-00-15](https://github.com/user-attachments/assets/cb38c6c2-dbd5-4f92-984b-03ba3e68acd2)

* After (20msec~)

![Screenshot from 2024-11-08 17-00-25](https://github.com/user-attachments/assets/cce4d647-f92c-4a2d-9bca-e77b28b61382)

* After, and I chaged `timeout_sec` to 0.01

![Screenshot from 2024-11-08 17-04-03](https://github.com/user-attachments/assets/076d54c3-72fd-40f3-b439-5d6f0baca98b)

In current AWSIM (v1.3.0) only the top LIDAR is published, so increasing `timeout_sec` also increases `sensor_points_delay_time_sec`.

The results of `autoware_ekf_localizer` are almost the same.

Just to be on the safe side, could you please leave the parameter at 0.01?

Reply:
Thanks for your analysis! I wasn’t aware that AWSIM only uses the top LiDAR. When I used AWSIM last Friday, I couldn't get any point cloud, so I measured the timestamp using the AWSIM lab.

First of all, if there’s only one LiDAR, it shouldn’t use the concatenate algorithm! :)
I'm not sure about the AWSIM plan—do you intend to use left and right LiDARs like in the AWSIM lab? If so, I think your recommendation to set it to 0.01 is a good approach for the current situation.

Reply:
Yes, I would ideally like to use 3 lidars in tier4/AWSIM, but I'm not sure when the 3 lidars will be implemented in tier4/AWSIM.

> First of all, if there’s only one LiDAR, it shouldn’t use the concatenate algorithm! :)

You are correct on this point, but we want to match the simulator configuration as closely as possible to the real car configuration, including topic names, node diagrams, etc.

Therefore, I would like to set the `timeout_sec` to 0.01 for the current situation.

Reply:
Thanks, fixed in https://github.com/tier4/awsim_sensor_kit_launch/pull/25/commits/01f20e84303601aedd7eb35fc8f028fcfe428752